src/main/java/io/nitric/faas/Faas.java

```diff
@@ -82,7 +82,7 @@
     public void startFunction(NitricFunction function) {
         Contracts.requireNonNull(function, "function");
 
-        // FIXME: Uncoverable code without mocking static methods (need to include Powermock)
+        // FIXME: Unrecoverable code without mocking static methods (need to include Powermock)
```

Comment:
This was a note on our inability to cover this code without use of powermock, probably just a spelling correction but 'Unrecoverable' is not correct.

Reply:
Opps thanks